modules/access-control/src/access-control-context-provider.tsx

```diff
@@ -105,79 +105,79 @@ export const AccessControlContext: FunctionComponent<PropsWithChildren<AccessCon
                     get(featureConfig, "branding.scopes.delete"),
```

Comment:
Better if can do optional chaining here as well instead of loadsh get

Reply:
Resolved by https://github.com/wso2/identity-apps/pull/2815/commits/0821946c8f541536e273de6495b2d9cf0daec665